packages/vscode-integration/src/common/glsp-vscode-connector.ts

```diff
@@ -56,6 +56,7 @@ interface ProgressReporter {
         message?: string | undefined;
         increment?: number | undefined;
     }>;
+    currentPercentage?: number | undefined;
```

Comment:
Minor: Sorry I missed this i the first review but declaring an optional property with an `undefined` union type is unnecessary.
```suggestion
    currentPercentage?: number;
```
Also applies to the other optional properties of `ProgressReporter`